### DISCUSSION TITLE
Welcome to Clobbr Discussions!

### DISCUSSION THREAD

**Discussion by danmindru** [parsecph/clobbr#153] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lucidhive** [comment]
hey @danmindru 
nice suite, I just got this one downloaded from setapp a few weeks ago but was not able to test it until now. 
seeing this CLI version from terminal makes me happier than the GUI version as I am on MacBookPro16" x86 Intel

which causes a strange anomaly when running the GUI version

I have 2 graphics cards in this machine and use an extension app called " gfxCardStatus v2,5" 

that alerts me which one is being used, the big AMD graphics card only ever did serve this system in adobe & finalcut apps during Rendering, but any & all browser or apps or dev tools run extreemly slow whilst the INTEL UHD 630 is smooth for some reason" 

I dont really know why this happens as the AMD is  supposed to be supporting the METAL3 =
AMD Radeon Pro 5500M:

  Chipset Model:	AMD Radeon Pro 5500M
  Type:	GPU
  Bus:	PCIe
  PCIe Lane Width:	x16
  VRAM (Total):	8 GB
  Vendor:	AMD (0x1002)
  Device ID:	0x7340
  Revision ID:	0x0040
  ROM Revision:	113-D3220E-190
  VBIOS Version:	113-D32206U1-019
  Option ROM Version:	113-D32206U1-019
  EFI Driver Version:	01.A1.190
  Automatic Graphics Switching:	Supported
  gMux Version:	5.0.0
  Metal Support:	Metal 3




However, just FYI, if you want the GUI Clobbr version working properly on all macs please make it use the Internal Intel 630 cards and turn of whatever is telling it to switch gx to AMD :-)


I am running Sonoma-Beta 14.3 Beta (23D5033f)

Best
M

**Reply by danmindru** [reply]
hey there!

Thanks so much for the report. This is a good one 😃
I've tried reproducing however it's hard to replicate this exact setup on a VM.

I think I could potentially use [a flag](https://www.electronjs.org/docs/latest/api/command-line-switches#--force_low_power_gpu) to force the internal GPU, not sure what consequences that would have. 
Do you think that would be worth trying?

I could perhaps get you a beta build somehow if you'd like to try it.

Cheers,
Dan
